requirements.txt

```diff
@@ -2,6 +2,7 @@ lit~=15.0
 # numpy 1.24 deprecates np.object, np.bool, np.float, np.complex, np.str,
 # and np.int which are used heavily in onnx-mlir.
 numpy~=1.22.2, <=1.23.5
+onnx~=1.15.0
```

Comment:
@brnorris03 Why did you add onnx to the requirement.txt file? We have a step in the docker file to install onnx here: https://github.com/onnx/onnx-mlir/blob/main/docker/Dockerfile.onnx-mlir-dev#L16 and locally we would need to do `pip install -e third_party/onnx` to ensure we are installing the correct onnx commit.

Reply:
It is required by the backend tests. I had been adding it manually to be able to test. I can change `requirements.txt` to use the local version. I don't really see a need to install manually.

Reply:
kk no worries, just curious! I will leave that decision to @AlexandreEichenberger :)